### PR TITLE
Additional jars for spark executor implementation

### PR DESCRIPTION
**Pull request by jozefbakus**
Additional jars for spark executor implementation
#71

### CODE REVIEW

File: src/main/resources/ui/controller/job/SparkController.js [hunk 1]
```diff
@@ -29,4 +29,14 @@ class SparkController {
...
     }
 
+    onDeleteAdditionalJar(oEv) {
+        console.log("onDeleteAdditionalJar");
+        UiListMethods.deleteListItem("/newJob/job/jobParameters/maps/additionalJars", this._model, oEv)
+    }
+
+    onAddAdditionalJar() {
+        console.log("onAddAdditionalJar");
+        UiListMethods.addListItem("/newJob/job/jobParameters/maps/additionalJars", this._model, "");
```
Comment (felipemmelo) [inline src/main/resources/ui/controller/job/SparkController.js:37]: Just for context on my side, will this method upload the JAR or is it just pointing at the given directory to list all jars available there?

**Review by kevinwallimann — APPROVED** [review]

**Review by felipemmelo — APPROVED** [review]

**Review by senelesithole — APPROVED** [review]

File: src/main/resources/ui/controller/job/SparkController.js [hunk 2]
```diff
@@ -29,4 +29,14 @@ class SparkController {
         UiListMethods.addListItem("/newJob/job/jobParameters/maps/appArguments", this._model, "");
     }
 
+    onDeleteAdditionalJar(oEv) {
+        console.log("onDeleteAdditionalJar");
+        UiListMethods.deleteListItem("/newJob/job/jobParameters/maps/additionalJars", this._model, oEv)
+    }
+
+    onAddAdditionalJar() {
+        console.log("onAddAdditionalJar");
```
Comment (Zejnilovic) [inline src/main/resources/ui/controller/job/SparkController.js]: Are the `console.log` supposed to stay like this? If yeas, why not add some more info to it?

File: src/main/resources/ui/controller/job/SparkController.js (hunk 1 above)
Comment (jozefbakus) [reply src/main/resources/ui/controller/job/SparkController.js:37]: It is just pointing at the given jar file. So you have to manually upload all required jars and give us permission to read them. As a future improvement, we can implement a component for uploading jars.

File: src/main/resources/ui/controller/job/SparkController.js (hunk 2 above)
Comment (jozefbakus) [reply src/main/resources/ui/controller/job/SparkController.js]: No, removed.

**Pull request merged**